File: python/10101.py
import sys

angle = []
for i in range(3):
    angle.append(int(sys.stdin.readline()))

if angle[0] == 60 and angle[1] == 60 and angle[2] == 60:
    print("Equilateral")
elif angle[0] + angle[1] + angle[2] == 180:
    if angle[0] == angle[1] or angle[0] == angle[2] or angle[1] == angle[2]:
        print("Isosceles")
    else:
        print("Scalene")
else:
    print("Error")

File: Baekjoon/10101.cpp
#include <iostream>
using namespace std;

int main() {
  ios::sync_with_stdio(0);
  cin.tie(0);
  cout.tie(0);
  
  int a,b,c;
  cin >> a >> b >> c;

  if (a == 60 && b == 60 && c == 60) {
    cout << "Equilateral" << '\n';
  } else if ( a + b + c == 180) {
    if (a == b || a == c || b == c) {
      cout << "Isosceles" << '\n';
    } else {
      cout << "Scalene" << '\n';
    }
  } else {
      cout << "Error" << '\n';
  }
  return 0;
}